lib/auth/init/init.ex

```diff
@@ -0,0 +1,126 @@
+defmodule Auth.Init do
+  @moduledoc """
+  `Init` as its' name suggests initializes the Auth Application 
+  by creating the necessary records in the various tables.
+
+  This is the sequence of steps that are followed to init the App:
+
+  1. Create the "Super Admin" person who owns the Auth App
+  based on the `ADMIN_EMAIL` environment/config variable.
+
+  > The person.id (1) for the Super Admin 
+  will own the remaining records so it needs to be created first. 
+
+  2. Create default records (Statuses & Roles)
+
+  3. Create the App and `AUTH_API_KEY` for the Auth App.
+  > Log the `AUTH_API_KEY` so that it can be exported.
+  """
+
+  require Logger
+  import Ecto.Changeset
+  alias Auth.{Person, Role, Repo, Status}
+
+  def main do
+    Logger.info("Initialising the Auth Database ...")
+    # check required environment variables:
+    Envar.is_set_all?(~w/ADMIN_EMAIL AUTH_URL ENCRYPTION_KEYS SECRET_KEY_BASE/)
```

Comment:
We could create the list of environment variables as a module variable to make it easier to change later on if required. However I think it does the job at the moment